app/models/user.rb

```diff
@@ -0,0 +1,21 @@
+class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
+
+  has_attached_file :avatar, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: "/avatar/thumb/user.png"
```

Comment:
Metrics/LineLength: Line is too long. [117/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)